.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -30,4 +30,5 @@ Add any other context or screenshots about the feature request here.
 - [ ] Code review and apply changes requested
 - [ ] Design test plan
 - [ ] QA
+- [ ] Add feature to Release Notes (if required)
```

Comment:
I would add this inside the "Development tasks" sublevel checks, the last one, or even like an additional check at the same level after "Development tasks". What do you think?

Reply:
Also, taking advantage of this PR, we could add a new check for "Implement unit tests (if needed)", so that we don't forget about them (we'll let UI tests stay for the moment).
Same, inside the "Development tasks" as a new check after `- [ ] ...` or as a new item at the same level. Personally, I would introduce both inside the "Development tasks" section, so that we englobe everything related to the development phase in one same item 👍

Reply:
Will add both to the Development tasks. So, it will look like:

```
- [ ] Development tasks
  - [ ] Implement whatever
  - [ ] ...
  - [ ] Implement unit tests (if needed)
  - [ ] Add feature to Release Notes (if required)
```